### ISSUE TITLE
Have your test in your data? 

### ISSUE THREAD

**Issue by Grandzxw** [epicjung/LVI_SAM_fixed#5]
And  the result is good?

**Comment by epicjung** [comment]
@Grandzxw Yeah. My result shows good performance. Could you tell me your sensor configuration?

**Comment by Grandzxw** [comment]
> @Grandzxw Yeah. My result shows good performance. Could you tell me your sensor configuration?
Can you share your data?Thank you!
